uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -142,10 +142,11 @@ sub IsValid {
     if ( ( $row[1] ) || ( $row[2] ) ) {
 
         $message =
-            "\n The Scan for the uploadID "
+            "\nThe Scan for the uploadID "
```

Comment:
@mohadesz,

To maintain consistency in the indentation across all the messages from the complete steps (whether displayed on screen as the preprocessing is launched from the back end, or in the notification spool table). This is purely aesthetic.